docs/machine-learning/basic-concepts-model-training-in-mldotnet.MD

```diff
@@ -0,0 +1,169 @@
+---
+title: Basic concepts for model training in ML.NET
+description: This article explains the basic concepts covering *model training* scenarios in ML.NET. Not all concepts are relevant to the more simple scenario of *prediction with existing model*.
+ms.date: 11/06/2018
+ms.topic: overview
+ms.custom: mvc
+#Customer intent: As a developer, I want to learn the basic ML.NET concepts related to model training so that I can understand how to leverage machine learning in my applications.
+---
+# Basic concepts for model training in ML.NET
+
+This article explains the basic concepts covering *model training* scenarios in ML.NET. Not all concepts are relevant to the more simple scenario of *prediction with existing model*.
+
+## Model training concepts
+
+This document covers the following concepts related to model training in ML.NET:
+
+- [*Data*](#data), represented as an `IDataView` interface.
```

Comment:
Throughout this section, the important types should be xref's

Reply:
Deadline constraints; will be handled later.